samples/samples-js/AddProductWithIdentityColumn/index.js

```diff
@@ -0,0 +1,14 @@
+module.exports = async function (context, req) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    const itemToInsert = {
+        "name": req.params?.name ?? "productWithIdentity",
+        "cost": req.params?.cost ?? 100
+    };
+    context.bindings.products = req.body ?? JSON.stringify(itemToInsert);
```

Comment:
Starting a new thread so it shows up correctly in the files view - for now let's keep it a priority to make these samples behave the same. Later on we can come back and rework all the checked in samples to add additoinal functionality such as parsing from the body. 

That won't be a huge amount of work - just change this line to 

`context.bindings.products = JSON.stringify(itemToInsert);`

(but also you should update `products` to `product` since there's only one object being added)